src/app/pages/checkout-payment/payment-concardis-creditcard-cvc-detail/payment-concardis-creditcard-cvc-detail.component.html

```diff
@@ -0,0 +1,16 @@
+<!-- error messages -->
+<div *ngIf="isCvcExpired()">
```

Comment:
ask for an IAD and adapt the ui to it, use localization keys instead of fixed string, add them to the en_us.properties file if necessary

Reply:
Done.